scripts/kconfig/guiconfig.py

```diff
@@ -0,0 +1,2310 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2019, Ulf Magnusson
```

Comment:
Please change to  `Nordic Semiconductor ASA and Ulf Magnusson`

Reply:
Fixed